tests/conftest.py

```
@@ -2429,6 +2429,13 @@ def snapshot_storage_class_name_scope_module(
     return [*storage_class_matrix_snapshot_matrix__module__][0]
 
 
+@pytest.fixture(scope="module")
+def snapshot_import_cron_format_storage_class_name_scope_module(
+    storage_class_matrix_snapshot_import_cron_format_matrix__module__,
+):
+    return [*storage_class_matrix_snapshot_import_cron_format_matrix__module__][0]
```

Ruff (0.16.0)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)